src/transaction.ts

```diff
@@ -92,6 +91,11 @@ export class Transaction {
      */
     public guardian: string;
 
+    /**
+     * The relayer, in address format, next version all the other addresses will not be string anymore.
```

Comment:
Can be rephrased.

```
The relayer address.
Note: in the next major version, `sender`, `receiver` and `guardian` will also have the type `Address`, instead of `string`.
```